README.md

```diff
@@ -61,15 +61,18 @@ import superjson from 'superjson';
 
 const jsonString = superjson.stringify({ date: new Date(0) });
 
-// jsonString === '{"json":{"date":"1970-01-01T00:00:00.000Z"},"meta":{"values":{date:"Date"}}}'
+// jsonString === '{"json":{"date":"1970-01-01T00:00:00.000Z"},"meta":{"values":{date:["Date"]}}}'
 ```
 
 And parse your JSON like so:
 
 ```js
+import superjson from 'superjson';
+
+const jsonString = '{"json":{"date":"1970-01-01T00:00:00.000Z"}}';
```

Comment:
`jsonString` is supposed to refer to the one in the code block above, so that change flips the meaning of this block entirely. Is there some way we can make this more explicit?

Reply:
I imagined it, but it was not clear to me as a user, I had to read and re-read to find it, so I thought it made more sense that way.

Perhaps putting both methods in the same code block cal/will make it clearer

Reply:
Sounds good!